roles/selinux_verify/vars/centosdev-7.yml

```diff
@@ -24,4 +26,3 @@ distro_files:
 
 distro_procs:
   - { key: 'docker', value: 'system_u:system_r:container_runtime_t:s0' }
-  - { key: 'docker-containerd', value: 'system_u:system_r:container_runtime_t:s0' }
```

Comment:
no more proc for docker-containerd?

Reply:
Nope, not included in this version of `docker`.  It will return once 1.13 is working again on CAHC